packages/cli/templates/default-backend-plugin/package.json.hbs

```diff
@@ -39,6 +38,7 @@
   },
   "devDependencies": {
     "@backstage/cli": "{{versionQuery '@backstage/cli'}}",
+    "@types/express": "{{versionQuery '@types/express' '4.17.13'}}",
```

Comment:
Express types are part of the public API of every backend plugin, which is why it needs to be in `dependencies`. This doesn't matter if you don't publish the package, but we might as well keep it in the correct place.